test/openbci-sdk-test.js

```diff
@@ -758,22 +586,16 @@ describe('#impedanceTesting', function() {
                 impedanceArray[1].channel.should.be.equal(2);
             });
             describe('#inputP', function() {
-                it('data array present',function() {
-                    impedanceArray[1].P.data.length.should.be.above(0);
-                });
-                it('valid average',function() {
-                    impedanceArray[1].P.should.have.property('average').above(-1);
+                it('got raw impedance value',function() {
```

Comment:
Ok so we say -1 here because when we create at the initial impedance array the proper `raw` (used to be `average`) is -1. So really the impedance will never be < 0, but this at least checks to make sure we changed the value. There could probably be a better test